### PR TITLE
readme: Merge download and install step into a command

### PR DESCRIPTION
**Pull request by supechicken**

### CODE REVIEW

**Comment by supechicken** [comment]
@googlebot I signed it!

File: README.md [hunk 1]
```diff
@@ -110,11 +110,10 @@ That's it! Surprised?
 crouton is a powerful tool, and there are a *lot* of features, but basic usage
 is as simple as possible by design.
 
-If you're just here to use crouton, you can grab the latest release from
-[https://goo.gl/fd3zc](https://goo.gl/fd3zc). Download it, pop open a shell
-(Ctrl+Alt+T, type `shell` and hit enter), make the installer executable with
-`sudo install -Dt /usr/local/bin -m 755 ~/Downloads/crouton`, then launch it
-with `sudo crouton` to see the help text. See the "examples" section for some
+If you're just here to use crouton, pop open a shell (Ctrl+Alt+T, type shell and hit enter),
+grab the latest release and make the installer executable with <br>
```
Comment (MrHallway0) [inline README.md:114]:
```suggestion
grab the latest release and make the installer executable with: <br>
```
Improve readability.

File: README.md [hunk 2]
```diff
@@ -142,17 +141,16 @@ start\* commands. Ta-da! That was easy.
...
-  2. Open a shell (Ctrl+Alt+T, type `shell` and hit enter)
-  3. Copy the installer to an executable location by running
-     `sudo install -Dt /usr/local/bin -m 755 ~/Downloads/crouton`
-  4. Now that it's executable, run the installer itself: `sudo crouton -t xfce`
-  5. Wait patiently and answer the prompts like a good person.
-  6. Done! You can jump straight to your Xfce session by running
+  1. Open a shell (Ctrl+Alt+T, type `shell` and hit enter)
+  2. Download the installer to an executable location by running
+     `sudo sh -c 'curl -L https://goo.gl/fd3zc -o /usr/local/bin/crouton && chmod +x /usr/local/bin/crouton'`
+  3. Now that it's executable, run the installer itself: `sudo crouton -t xfce`
```
Comment (MrHallway0) [inline README.md:147]:
```suggestion
  3. Now that it's executable, run the installer itself: `sudo crouton -t xfce -r sid`
```
Update command to use `sid`.

**Review by MrHallway0 — CHANGES_REQUESTED** [review]
@supechicken thank you for supporting Crouton!
I have found two things that could be used with changes, other than these things I think your pull request is good.

File: README.md (hunk 1 above)
Comment (supechicken) [reply README.md]: Thanks for the suggestion, change made :)

**Review by MrHallway0 — APPROVED** [review]
Looks good!

**Comment by DennisLfromGA** [comment]
My opinion, the crouton author's original wording is best, I wouldn't change it.

-**DennisLfromGA**

**Comment by dnschneid** [comment]
I appreciate the suggestions (and thanks for creating a PR), but the README intentionally avoids magic one-liners as I feel it's important to educate users as to what is going on.  Using crouton is as much a learning experience for new users as it is a tool for the linux-savvy.

**Pull request closed without merging**